indico/modules/events/editing/client/js/components/FileManager/selectors.js

```diff
@@ -0,0 +1,5 @@
+export const getFiles = state => {
+  const files = {};
+  state.fileTypes.forEach(fileType => (files[fileType.id] = fileType.files.map(file => file.uuid)));
```

Comment:
`=` assignment does not belong in an arrow function without curly braces. Doing an assignment as an expression is pretty ugly in this context..

```js
export const getFiles = state => {
  const files = {};
  state.fileTypes.forEach(fileType => {
    files[fileType.id] = fileType.files.map(file => file.uuid);
  });
  return files;
};
```